src/Infrastructure/LeanCode.PdfRocket/PdfRocketGenerator.cs

```diff
@@ -0,0 +1,104 @@
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using LeanCode.ViewRenderer;
+
+namespace LeanCode.PdfRocket
+{
+    public class PdfRocketGenerator
+    {
+        public const string ApiUrl = "https://api.html2pdfrocket.com/";
+        private readonly Serilog.ILogger logger = Serilog.Log.ForContext<PdfRocketGenerator>();
+
+        private readonly PdfRocketConfiguration config;
+        private readonly IViewRenderer viewRenderer;
+        private readonly HttpClient client;
+
+        public PdfRocketGenerator(PdfRocketConfiguration config, IViewRenderer viewRenderer, HttpClient client)
+        {
+            this.config = config;
+            this.viewRenderer = viewRenderer;
+            this.client = client;
+        }
+
+        public virtual Task<Stream> GenerateFromHtml(
```

Comment:
`Async`, when we're at it.

Reply:
IMO there's no benefit in preserving this one line in a stack trace, but whatever.